src/core/lib/gprpp/arena.cc

```diff
@@ -58,7 +58,21 @@ Arena* Arena::Create(size_t initial_size) {
        GPR_CACHELINE_SIZE % GPR_MAX_ALIGNMENT == 0)
           ? GPR_CACHELINE_SIZE
           : GPR_MAX_ALIGNMENT;
-  return new (gpr_arena_malloc<alignment>(alloc_size)) Arena(initial_size);
+  return gpr_arena_malloc<alignment>(alloc_size);
+}
+
+Arena* Arena::Create(size_t initial_size) {
+  return new (ArenaStorage(initial_size)) Arena(initial_size);
+}
+
+void* Arena::CreateWithAlloc(size_t initial_size, size_t alloc_size,
```

Comment:
since this is c++11 now, I'd return a `pair<Arena*, void*>`

Reply:
Done